R/utils.R

```diff
@@ -1,29 +1,4 @@
-
-
-merge.list <- function(x,y,..., open=FALSE,
-                       warn=FALSE, context="object") {
-  
-  y <- as.list(y)
-  
-  ## Merge two lists
-  common <- intersect(names(x), names(y))
-  
-  x[common] <- y[common]
-  
-  if(open)  {
-    nw <- !is.element(names(y),names(x)) #| names(y) == wild
-    x <- c(x,y[nw])
-  } else {
-    if(length(common)==0 & warn) {
-      warning(
-        paste0("Found nothing to update: ", context),
-        call.=FALSE
-      )
-    }
-  }
-  x
-}
-
+# Replaces old merge.list(strict = TRUE) functionality
```

Comment:
I didn't follow what this `strict=TRUE` comment for `combine_list` is about.  I don't spot a `strict` argument on the `merge.list` removed above or mrgsolve's variant.

Reply:
Sorry ... it was `open = FALSE` ... open used to mean that you could add to the list through the merge. It was a long time ago and long ago mrgsolve evolved so that the list was fixed on compile and you could only update was already there.